src/app/stack/overview/chart-component.ts

```diff
@@ -25,7 +25,16 @@ import * as c3 from 'c3';
     template: ``,
     encapsulation: ViewEncapsulation.None,
     inputs: ['data', 'chartOptions', 'configs'],
-    styles: [`.ng2-c3{ display:block;}`] // This is required for proper positioning of tooltip
+    styles: [
+        `.ng2-c3 {
+            display:block; // This is required for proper positioning of tooltip
+        }
+        f8-chart .c3-chart-arcs-title {
```

Comment:
This need not be generic I guess. This can be handled in the individual component level.